src/main/java/baritone/process/elytra/ElytraBehavior.java

```diff
@@ -159,9 +164,11 @@ public void tick() {
                 this.ticksNearUnchanged = 0;
             }
 
-            // Obstacles are more important than an incomplete path, handle those first.
-            this.pathfindAroundObstacles();
-            this.attemptNextSegment();
+            if (ctx.player().position().y < context.maxHeight && ctx.player().position().y > 0) {
```

Comment:
This looks like comparing world coordinates against a range of zero based coordinates?

Reply:
fixed